cmd/gatekeeper-test-alpha/gatekeeper-test-alpha.go

```diff
@@ -1,16 +1,54 @@
 package main
 
 import (
+	"errors"
 	"fmt"
 	"os"
 
+	"github.com/open-policy-agent/gatekeeper/pkg/gktest"
 	"github.com/spf13/cobra"
 )
 
+var run string
+
+func init() {
+	rootCmd.Flags().StringVarP(&run, "run", "r", "",
+		`regular expression which filters tests to run by name`)
+}
+
 var rootCmd = &cobra.Command{
 	Use:   "gatekeeper-test-alpha path [--run=name]",
 	Short: "Gatekeeper Test Alpha is a unit test CLI for Gatekeeper Constraints",
+	Example: `  gatekeeper-test-alpha label-tests.yaml
+  gatekeeper-test-alpha tests/... --run forbid-labels`,
```

Comment:
Perhaps we can include several examples to give users a hint at the regex syntax accepted and whether we're going to expect patterns to be anchored or not. For example, `kubectl` gives several commented examples:

```
$ kubectl get -h
...
Examples:
  # List all pods in ps output format.
  kubectl get pods

  # List all pods in ps output format with more information (such as node name).
  kubectl get pods -o wide

  # List a single replication controller with specified NAME in ps output format.
  kubectl get replicationcontroller web
...
```

Reply:
Good idea. Done.